config/set/nette-31.php

```diff
@@ -0,0 +1,130 @@
+            'Nette\Database\Context' => 'Nette\Database\Explorer',
+            'Nette\Database\IRow' => 'Nette\Database\Row',
+            'Nette\Database\IRowContainer' => 'Nette\Database\ResultSet',
+            'Nette\Database\Table\IRow' => 'Nette\Database\Table\ActiveRow',
+            'Nette\Database\Table\IRowContainer' => 'Nette\Database\Table\Selection',
+
+            // https://github.com/nette/forms/compare/v3.0.7...v3.1.0-RC2
+            'Nette\Forms\IControl' => 'Nette\Forms\Control',
+            'Nette\Forms\IFormRenderer' => 'Nette\Forms\FormRenderer',
+            'Nette\Forms\ISubmitterControl' => 'Nette\Forms\SubmitterControl',
+
+            // https://github.com/nette/mail/compare/v3.0.1...v3.1.5
+            'Nette\Mail\IMailer' => 'Nette\Mail\Mailer',
+
+            // https://github.com/nette/security/compare/v3.0.5...v3.1.2
+            'Nette\Security\IAuthorizator' => 'Nette\Security\Authorizator',
+            'Nette\Security\Identity' => 'Nette\Security\SimpleIdentity',
+            'Nette\Security\IResource' => 'Nette\Security\Resource',
+            'Nette\Security\IRole' => 'Nette\Security\Role',
+
+            // https://github.com/nette/utils/compare/v3.1.4...v3.2.1
+            'Nette\Utils\IHtmlString' => 'Nette\HtmlStringable',
+            'Nette\Localization\ITranslator' => 'Nette\Localization\Translator',
+
+            // https://github.com/nette/latte/compare/v2.5.5...v2.9.2
+            'Latte\ILoader' => 'Latte\Loader',
+            'Latte\IMacro' => 'Latte\Macro',
+            'Latte\Runtime\IHtmlString' => 'Latte\Runtime\HtmlStringable',
+            'Latte\Runtime\ISnippetBridge' => 'Latte\Runtime\SnippetBridge',
+        ],
+    ]]);
+
+    $services->set(RenameMethodRector::class)->call('configure', [[
+        RenameMethodRector::METHOD_CALL_RENAMES => ValueObjectInliner::inline([
+            // https://github.com/nette/caching/commit/60281abf366c4ab76e9436dc1bfe2e402db18b67
+            new MethodCallRename('Nette\Caching\Cache', 'start', 'capture'),
+            // https://github.com/nette/forms/commit/faaaf8b8fd3408a274a9de7ca3f342091010ad5d
+            new MethodCallRename('Nette\Forms\Container', 'addImage', 'addImageButton'),
+            // https://github.com/nette/utils/commit/d0427c1811462dbb6c503143eabe5478b26685f7
+            new MethodCallRename('Nette\Utils\Arrays', 'searchKey', 'getKeyOffset'),
+        ]),
+    ]]);
+
+    $services->set(RenameStaticMethodRector::class)
+        ->call('configure', [[
+            RenameStaticMethodRector::OLD_TO_NEW_METHODS_BY_CLASSES => ValueObjectInliner::inline([
+                // https://github.com/nette/utils/commit/8a4b795acd00f3f6754c28a73a7e776b60350c34
+                new RenameStaticMethod('Nette\Utils\Callback', 'closure', 'Closure', 'fromCallable'),
+            ]),
+        ]]);
+
+    // TODO change $router[] = new Router() to $router->addRoute() because of deprecated flags
```

Comment:
Something like `DimFetchAssignToMethodCallRector`? :) 

We don't have such a rule yet, but should be easy to add.
This might be a good start: https://github.com/rectorphp/rector/blob/master/rules/magic-disclosure/src/Rector/Assign/GetAndSetToMethodCallRector.php

Reply:
Tomorrow I will try to create something :)

Reply:
I didn't write generic rule for now. Just one for this use case inspired by ArrayAccessSetControlToAddComponentMethodCallRector